app/views/components/_glance-metric.html.erb

```diff
@@ -1,20 +1,25 @@
 <%
   name ||= false
-  figure ||= false
+  figure ||= nil
   period ||= false
   context ||= ""
   trend_percentage ||= ""
   measurement_display_label ||= ""
   measurement_explicit_label ||= ""
 %>
-<% if name && figure && period %>
+<% if name && period %>
   <div class="app-c-glance-metric govuk-body">
     <h3 class="app-c-glance-metric__heading"><%= name %></h3>
-    <span class="app-c-glance-metric__figure"><%= figure %><% if measurement_display_label %><span class="app-c-glance-metric__measurement"
-        <% if measurement_explicit_label %>
-          aria-label="<%= measurement_explicit_label %>"
-        <% end %>><%= measurement_display_label %></span>
-    <% end %>
+
+      <% if figure.nil? %>
+      <span class="app-c-glance-metric__figure">No data</span>
```

Comment:
I might have added the class name to app-c-glance-metric__figure--no-data and added a rule in the CSS to reduce the size of the font to make it a little tidier, but I'm not super-bothered about this, seeing this version of the page at all is going to be an edge case and I'm not sure it's worth adding to the CSS for it.